UnityAds/src/main/java/com/mopub/mobileads/UnityInterstitial.java

```diff
@@ -45,22 +47,45 @@ protected void load(@NonNull final Context context, @NonNull final AdData adData
         mContext = context;
 
         setAutomaticImpressionAndClickTracking(false);
-        UnityAds.load(mPlacementId);
 
         mUnityAdsAdapterConfiguration.setCachedInitializationParameters(context, extras);
 
         UnityRouter.getInterstitialRouter().addListener(mPlacementId, this);
```

Comment:
do not need to add this listener anymore.

Reply:
@jindoujiao this is still needed for extended listener like `onUnityAdsClick`